### PR TITLE
New blog post showing real applications running on Flex

### PR DESCRIPTION
**Pull request by cloudnull**
This change updates our blog to talk about real applications running on Flex; applications that make use of our Open Infrastructure native capabilities.

### CODE REVIEW

**Comment by busterswt** [comment]
LGTM!

**Review by sudhakar-datta — APPROVED** [review]
LGTM

File: docs/blog/posts/2024-08-21-creating-ghost-site-using-flex.md [hunk 1]
```diff
@@ -54,7 +56,7 @@ Connect the subnet to our flex-router
 openstack --os-cloud {cloud_name} router add subnet flex-router flex-subnet
 ```
 
-Now we need to create our security group, this is the group which specify the network access rules. For our example now we are only going to allow SSH access. 
+Now we need to create our security group, this is the group which specify the network access rules. For our example now we are only going to allow SSH access.
```
Comment (sulochan) [inline docs/blog/posts/2024-08-21-creating-ghost-site-using-flex.md:59]: Should be: For our example we are only going to allow SSH access. ? just a nitpick.

Comment (cloudnull) [reply docs/blog/posts/2024-08-21-creating-ghost-site-using-flex.md]: done.

**Review by sulochan — APPROVED** [review]

**Review by timolow — APPROVED** [review]

**Pull request merged**